src/_includes/alert.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-light {{ include.display }}" role="alert">
+  <div class="container fs-7">
+    DGS has issued a Draft RFP for GTFS-Realtime software so public transit agencies can provide riders with real time fleet
```

Comment:
@mrose914 I think "real time fleet location" should be `real-time` here b/c it's an adjective . Just wanted to double-check you wanted this to be "real time". 

On MobiMart, the only time there are no hyphens between `real-time` is when it is used in the phrase, "in real time": https://github.com/search?q=repo%3Acal-itp%2Fmobility-marketplace+%22real+time%22&type=code

Otherwise, when `real-time` is describing a location or kind of data, it's hyphenated: https://github.com/search?q=repo%3Acal-itp%2Fmobility-marketplace+%22real-time%22&type=code